extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [UNRELEASED]
```

Comment:
You can remove the "unreleased" header here!

Reply:
Thanks, I'll remove it